spec-main/package.json

```diff
@@ -16,6 +16,6 @@
   "dependencies": {
     "chai-as-promised": "^7.1.1",
     "dirty-chai": "^2.0.1",
-    "pdfjs-dist": "^2.2.228"
+    "pdfjs-dist": "^2.3.200"
```

Comment:
We only use pdfjs to read the number of pages in a pdf generated by a test, so I'm not sure updating this is really useful.

Besides that, even if we do update, we should probably update to latest (2.5.207).

Reply:
I see - I thought this was being used to display PDFs. I guess that happens inside chromium then?

Reply:
That's correct, Chromium uses pdfium to display pdf. pdfjs is not present in the released version of Electron.

Reply:
As this is in spec-main, seems simple enough to confirm: remove the dep and see if CI is still happy

Reply:
@ckerr we use the dep (a grep confirms this). It is only used in one test, to count pages in a printed PDF from the `printToPDF` API.